composer.json

```diff
@@ -129,6 +129,7 @@
             "phpspec run --no-code-generation"
         ],
         "migrate" : [
+            "@verify",
```

Comment:
@tuxpiper I think this is what you were wondering about with migrate no? 
The only thing is that while this runs the verification, it doesn't *stop* the migration from happening if it fails, it reports the failure but it still attempts the migration no matter what. Not really sure how to do that, or if we want to do that (I suppose we would if we figure it out)

Reply:
oops sorry, this flied under the radar.

I would rather have a verification error be fatal, but if composer is not willing to help with the flow control, then I think it's fine. At least there will be some additional error output that may help debug these problems better.

Reply:
I figured out how to do it. It's less neat in errors because it shows an exception but I believe it's still better since it allows to stop the migration and error out instead of pretending things are ok :D. If you agree with this I will push and we can get it merged. 
<img width="1413" alt="Screen Shot 2019-09-04 at 13 12 16" src="https://user-images.githubusercontent.com/2434401/64272418-b46fcf00-cf15-11e9-9026-e0e76db8c765.png">

Reply:
basically just thows a generic exception at the end of the execution in the CLI (only in CLI) if there are any failures to indicate it failed

Reply:
This is the awesomesauce!  👍  Where is :partyparrot: 😄 ?